cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/BalancingConstraint.java

```diff
@@ -110,6 +114,7 @@ Properties setProps(Properties props) {
     props.put(AnalyzerConfig.TOPICS_WITH_MIN_LEADERS_PER_BROKER_CONFIG, _topicsWithMinLeadersPerBrokerPattern.pattern());
     props.put(AnalyzerConfig.MIN_TOPIC_LEADERS_PER_BROKER_CONFIG, Integer.toString(_minTopicLeadersPerBroker));
     props.put(AnalyzerConfig.FAST_MODE_PER_BROKER_MOVE_TIMEOUT_MS_CONFIG, Long.toString(_fastModePerBrokerMoveTimeoutMs));
+    props.put(AnalyzerConfig.DEFAULT_BROKER_SET_RESOLVER_CLASS, _brokerSetResolver.getClass().getName());
```

Comment:
Why do we have a "DEFAULT" in the var name here? I think BROKER_SET_RESOLVER_CLASS makes more sense to me.

Reply:
Fixed